### PR TITLE
Feat/423 utilize enter trigger

### PR DESCRIPTION
**Pull request by Sm00g15**
Enter key now triggers below buttons:

1 - LoginCreate - Create New Wallet button
2 - LoginImport - Open button
3 - AddDelegateModal - Delegate button
4 - SendConfirmationModal - Confirm button
5 - ChangeDelegateModal - Change Delegate button

### CODE REVIEW

**Comment by Sm00g15** [comment]
The event.detail === 0 portion of this change is to prevent the openFile and saveFile functions from running unless they're summoned by a click event strictly. I did this because originally when the enter button was pressed to submit it was also triggering the file opener and file saver.

File: app/containers/LoginImport/index.js [hunk 1]
```diff
@@ -119,13 +121,20 @@ class LoginImport extends Component<Props> {
     this.setState({isShowed: !this.state.isShowed});
   }
 
+
+  onEnterPress = (event) => {
+    if(this.state.password !== "" && this.state.walletFileName !== "" && this.state.walletLocation !== "" && !this.state.isLoading && event.key === "Enter") {
```
Comment (developer0623) [inline app/containers/LoginImport/index.js]: @Sm00g15 Please use destructuring assignment.

File: app/containers/LoginCreate/index.js [hunk 2]
```diff
@@ -270,6 +282,7 @@ class LoginCreate extends Component<Props> {
       !this.state.isPasswordValidation ||
```
Comment (developer0623) [inline app/containers/LoginCreate/index.js]: Please use destructuring assignment

File: app/components/AddDelegateModal/index.js [hunk 3]
```diff
@@ -333,6 +339,7 @@ class AddDelegateModal extends Component<Props> {
     } = this.getBalanceState(balance, amount, t);
     return (
       <Modal
+        onKeyDown={this.onEnterPress}
```
Comment (developer0623) [inline app/components/AddDelegateModal/index.js]: {(event)=> this.onEnterPress(event.key, isDisabled)}

......

onEnterPress = (keyVal, isDisabled)=> {
if (keyVal === 'Enter' && !isDisabled) {
....
}
}

File: app/containers/LoginCreate/index.js (hunk 2 above)
Comment (Sm00g15) [reply app/containers/LoginCreate/index.js]: Code has been updated

**Comment by umurb** [comment]
1. Change the color of the buttons to :hover color after Enter key is pressed.
screen shot 2018-10-25 at 8 09 54 am


2. Add Enter Key functionality to below screen
screen shot 2018-10-25 at 7 55 19 am

3. Add Enter Key functionality to below screen (enter should trigger the button only after 15th word)
screen shot 2018-10-25 at 8 05 34 am

**Comment by umurb** [comment]
- [ ] As you mentioned in Riot, AddDelegateModal is not working.

- [ ] Disabling the button works and looks great! I have only one request with `Create New Wallet` scene. There is a 1 sec delay in between me hitting the enter button, and wallet going into `Loading` state, where the button gets disabled. Is it possible to disable the button right after user hitting enter?

- [ ] Enter button still not working in this scene:

screen shot 2018-10-26 at 9 52 11 am

**Comment by Sm00g15** [comment]
In the code I've pushed up the enter trigger can only be utilized when an input (text or file) is in focus, however I've just learned that you can add an event listener to the entire document so that the user will be able to trigger an event with enter whether they're in, or out of, focus. I can add this to the code if you'd like.

**Comment by umurb** [comment]
Everything seems to be working fine. I think adding an overall event listener at this point would be too soon. Let's not add them in this upcoming release and test it out as is, and think of upsides and downsides of it a little more.

@developer0623 I've tested the UI, and everything is working fine. Can you review the latest commits?

**Pull request merged**